tilp/trunk/build/scripts/install_tilp.sh

```diff
@@ -249,9 +253,9 @@ fi
 if [ "x$UNAME" = "xDarwin" ]; then
     echo "Modifying PKG_CONFIG_PATH on MacOS X"
     if [ "x$PKG_CONFIG_PATH" = "x" ]; then
-        PKG_CONFIG_PATH="/usr/lib/pkgconfig:/usr/local/lib/pkgconfig:$PREFIX/lib/pkgconfig:/usr/local/opt/libarchive/lib/pkgconfig:/usr/local/opt/libffi/lib/pkgconfig:/opt/X11/lib/pkgconfig"
+        PKG_CONFIG_PATH="/opt/homebrew/lib/pkgconfig:/opt/homebrew/opt/libarchive/lib/pkgconfig:/opt/homebrew/opt/libffi/lib/pkgconfig:/opt/X11/lib/pkgconfig"
     else
-        PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/usr/lib/pkgconfig:/usr/local/lib/pkgconfig:$PREFIX/lib/pkgconfig:/usr/local/opt/libarchive/lib/pkgconfig:/usr/local/opt/libffi/lib/pkgconfig:/opt/X11/lib/pkgconfig"
+        PKG_CONFIG_PATH="$PKG_CONFIG_PATH:/opt/homebrew/lib/pkgconfig:/opt/homebrew/opt/libarchive/lib/pkgconfig:/opt/homebrew/opt/libffi/lib/pkgconfig:/opt/X11/lib/pkgconfig"
```

Comment:
Same here, although I guess you can also leave `/usr/lib/pkgconfig` as is, users may have their own things there, regardless of brew.

Reply:
Issue with this is that some users have old/legacy homebrew installations that use the directory, and including it overrides the arm64 versions for x86, which ld ignores, leading to issues with undefined symbols. Possibly could it be moved to the back to lower its priority?

Reply:
Well either way it's not going to create more issues than right now, as it's going to be biased towards one or the other. Maybe it's time to prioritize the arm/new path first?

Reply:
It would be relatively easy to detect Arm vs x86, I can probably implement that check relatively fast.

Reply:
Sure, but still prefer using `$(brew --prefix)` for the brew-specific paths.